tests/base.py

```diff
@@ -173,6 +173,13 @@ def setUp(self):
             structure.save()
             db_session.commit()
 
+    def get_admin_location_id(self):
+        payload = jwt.decode(ADMIN_TOKEN, verify=False)
+        email = payload['UserInfo']['email']
+        user = User.query.filter_by(email=email).first()
+        location = Location.query.filter_by(name=user.location).first()
```

Comment:
why are you equating name to user location?

Reply:
That is the name in the location model